src/main/java/uk/gov/justice/digital/delius/service/RequirementService.java

```diff
@@ -38,27 +38,17 @@
 
     public ConvictionRequirements getRequirementsByConvictionId(String crn, Long convictionId, boolean activeOnly) {
 
-        Predicate<Requirement> activeFilter = activeFilter(activeOnly);
-        var requirements = Optional.of(getEvent(crn, convictionId))
+        var requirements = Optional.of(getEvent(crn, convictionId, activeOnly))
```

Comment:
👎 
The activeOnly filter must be applied to requirements. We can try to additionally apply it to events and disposals to see if it helps get round this problem but it must be on requirements - that's what we are relying on the method to do. Otherwise we (and others?) will start getting inactive requirements again